.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -4,44 +4,111 @@ about: Use this template to track all release checks for your component
+**Version to be included in Eclipse Tractus-X release**:
+<!-- Responsibility: Committer -->
 
-This issue tracks all checks, that need to be performed for a product release in Eclipse Tractus-X.
+- App version: *version placeholder*
+- Helm Chart version: *version placeholder*
 
-- [ ] **Interoperability** checks performed
-- [ ] **Data Sovereignty** checks performed
+**Leading product repository:** *repository link*
 
-# General Checks
+## Compliance
+<!-- Responsibility: Committer, ExpertGroup, Technical Committee for Standardization -->
 
+- [ ] Possible changes (through the new features) on related Catena-X standards are considered and addressed
 - [ ] [**Tractus-X Release Guidelines**](https://eclipse-tractusx.github.io/docs/release)(TRGs) fulfilled
 
 Make sure to open and fill in a separate **documentation issue** in your product repository using the [Quality Gate Checklist issue template](https://github.com/eclipse-tractusx/.github/blob/main/.github/ISSUE_TEMPLATE/qg-checklist.md)
 
 > [!NOTE]
 > Note: most criteria for documentation and security are now covered in TRGs
 
-# Security Checks
+## Functionality
+<!-- Responsibility: Committer, Testmanagement -->
+
+- [ ] Feature works as expected and described
+- [ ] Backward compatibility maintained (depending Major/ Minor) 
+
+## Performance
+<!-- Responsibility: Testmanagement -->
+
+- [ ] Memory and CPU usage is within acceptable limits oriented with goldilocks 
+
+## Testing
+<!-- Responsibility: Committer, Testmanagement -->
+
+- [ ] Unit tests cover all new features
+- [ ] Integration tests are updated
+- [ ] E2E/Integration test passed
+- [ ] Regressions tests passed
+- [ ] User Journey approved
+
+## Feature summary
+<!-- 
+Responsibility:
+- Committer -> adding the features
+- Testmanagement -> approval for teststatus
+-->
+
+Please provide a list of all features that have been developed in the current phase. This list is essential for several reasons:
+
+- **Test Preparation:** Identifying new features will help in the creation of new end-to-end (E2E) tests and ensure that the relevant aspects are thoroughly tested.
+- **Awareness:** It will raise awareness of the features that need to be prioritized during the testing process.
+- **Board Management:** This will assist in keeping the project board organized and up-to-date.
+
+By documenting these features, you will contribute to a smoother and more efficient release process.
+
+> [!NOTE]
+> Note: all features needs to be closed to get approval for release
+
+| Feature | Test Status | Note |
+|----------|----------|----------|
+| linked feature 1 | Test status | note 1 |
+| linked feature 2 | Test status | note 2 |
+| linked feature 3 | Test status | note 3 |
+
+## Documentation
+<!-- Responsibility: Committer -->
+
+- [ ] Documentation (related to new implemented features e.g. migration, api) is updated to reflect changes
+- [ ] README and other project documentation are clear and up-to-date
+
+## Deployment Readiness
```

Comment:
the name - which is referring to deployment - doesn't make sense to me considering bellows
```suggestion
## Release Documentation
```